packages/@livestore/common/src/schema/EventDef.ts

```diff
@@ -191,6 +191,8 @@ export type Materializer<TEventDef extends EventDef.AnyWithoutFn = EventDef.AnyW
     eventDef: TEventDef
     /** Can be used to query the current state */
     query: MaterializerContextQuery
+    /** The client ID that created this event */
+    clientId: string
```

Comment:
Now that I think about it, wouldn't make it more sense to pass in the full `LiveStoreEvent` here instead?